apps/server/src/modules/task-card/controller/dto/task-card.response.ts

```diff
@@ -19,6 +20,11 @@ export class TaskCardResponse {
 	})
 	id: string;
 
+	@ApiProperty({
```

Comment:
add `@DecodeHtmlEntities()`?

Reply:
no, we should not allow titles to store html in the first place